<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="TINTN003DKVT9KM" name="KOIMSO-12" tests="1" failures="1" timestamp="19 Nov 2015 11:28:18 GMT" time="105.503" errors="0">
  <testcase name="TEO_XML" time="100.012" classname="msol.XML_Validations.KOIMSO_12">
    <failure type="java.lang.Exception" message="The test case is failed either because of no created record or because of Duplicate PO">
      <![CDATA[java.lang.Exception: The test case is failed either because of no created record or because of Duplicate PO
	at msol.XML_Validations.KOIMSO_12.TEO_XML(KOIMSO_12.java:121)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- TEO_XML -->
</testsuite> <!-- KOIMSO-12 -->
